ci/models.py

```diff
@@ -53,6 +54,7 @@ class JobStatus(object):
         (FAILED_OK, "Allowed to fail"),
         (CANCELED, "Canceled by user"),
         (ACTIVATION_REQUIRED, "Requires activation"),
+        (INTERMITTENT_OK, "Intermittent Failure"),
```

Comment:
Everywhere else you say intermittent_ok, but here you say failure. Perhaps we should be consistent? I think I like intermittent failure everywhere since to me that suggests that it's OK more than it's not.

Reply:
I was following Brian's paradigm when he dealt with failed_ok. Consider the following:

```python
if INTERMITTENT_FAILURE:
  do success code
```

Is the above less confusing than:

```python
if INTERMITTENT_OK:
  do success code
```